src/Model/Article.vala

```diff
@@ -157,33 +160,40 @@ public class FeedReader.Article : GLib.Object {
 		var date_day = m_date.get_day_of_year();
 		var date_week = m_date.get_week_of_year();
 
-		string time = (addTime) ? ", %H:%M" : "";
-
-		if(date_year == 1900)
-		{
-			//return _("no date available");
-		}
-		else if(date_year == now_year)
+		var formats = new Gee.ArrayList<string>();
+		if(date_year == now_year)
 		{
 			if(date_day == now_day)
 			{
-				return m_date.format("%H:%M");
+				addTime = true;
 			}
-			else if(date_day == now_day-1)
+			else if(date_day == now_day -1)
 			{
-				return _("Yesterday") + m_date.format(", %H:%M");
+				formats.add("Yesterday");
```

Comment:
Is this still translatable, or am I missing something here?

Reply:
Good catch, I didn't mean to change that :\